apps/docs/content/components/FormControl.mdx

```diff
@@ -328,6 +328,8 @@ Pass the `fullWidth` prop to `FormControl` to provide additional behavior and st
 
 Pass the `required` prop to `FormControl` to provide additional behavior and state context to all `children`, rather than the input only.
 
+When marking a field as required, it is recommended to also provide a corresponding message at the start of the form informing the user that "_all fields marked with an asterisk (\*) are required_".
```

Comment:
Nice!

Could we also update the form example [further up the page please](https://primer-be045554d2-26139705.drafts.github.io/brand/components/FormControl#layout)?

![Screenshot 2024-11-15 at 16 26 11](https://github.com/user-attachments/assets/2597b51d-c61b-432f-94bc-7297ae4f6e73)

Reply:
Ah good spot, thanks.

I'd searched for all `<form>` tags but looks like that was missing one. I've wrapped it in one now and made the required change 👍

Reply:
Thanks @joshfarrant.

@simmonsjenna here's a sample of the required field statement we need Site to add to all form designs going forward. Let us know if you have any further guidance on spacing, positioning or colors. We can fix in a follow up. 

![Screenshot 2024-11-21 at 10 45 15](https://github.com/user-attachments/assets/a928693f-0a34-476e-9cd8-18820ffcace0)

[Link](https://primer-be045554d2-26139705.drafts.github.io/brand/components/FormControl#layout)

Reply:
@rezrah LGTM